src/plone/restapi/tests/http-examples/vocabularies_get_filtered_by_token.resp

```diff
@@ -3,19 +3,6 @@ Content-Type: application/json
 
 {
   "@id": "http://localhost:55001/plone/@vocabularies/plone.app.vocabularies.ReallyUserFriendlyTypes?token=doc", 
-  "items": [
-    {
-      "title": "DX Test Document", 
-      "token": "DXTestDocument"
-    }, 
-    {
-      "title": "Page", 
-      "token": "Document"
-    }, 
-    {
-      "title": "Test Document", 
-      "token": "ATTestDocument"
-    }
-  ], 
-  "items_total": 3
+  "items": [], 
```

Comment:
@sneridagh crap. I liked this example because with a query by token and title we got different results. :)

We might have to amend the example here to make sure we show some results. What do you think?

Reply:
mmm true, it's ugly this way. I will take a look.

Reply:
FYI: I fixed the example.